src/plugins/console/public/application/containers/main/main.tsx

```diff
@@ -51,19 +60,41 @@ export function Main({ isEmbeddable = false }: MainProps) {
         <EuiFlexItem>
           <EuiSplitPanel.Outer grow={true} borderRadius={isEmbeddable ? 'none' : 'm'}>
             <EuiSplitPanel.Inner grow={false} className="consoleTabs">
-              <TopNavMenu
-                disabled={!done}
-                items={getTopNavConfig({
-                  selectedTab,
-                  setSelectedTab,
-                })}
-              />
+              <EuiFlexGroup direction="row" alignItems="center" gutterSize="s">
+                <EuiFlexItem>
+                  <TopNavMenu
+                    disabled={!done}
+                    items={getTopNavConfig({
+                      selectedTab,
+                      setSelectedTab,
+                    })}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiButtonIcon iconType="save" onClick={() => {}} />
```

Comment:
I think we should add aria labels here otherwise will be quite hard to understand what these are with a screen reader. Also I was thinking perhaps we could add a little tooltip with a description of the action? Otherwise might be hard to understand for the user what these buttons do 🤔 cc: @julianrosado

Reply:
Good call about the aria labels! Added them with [d9afe73](https://github.com/elastic/kibana/pull/190820/commits/d9afe73f98ba8c383fdd5abb460f8c882d08902f).

Yes, I think adding tooltips is a great idea! Let's see if @julianrosado has any thoughts on this.

Reply:
That's a _great_ idea @sabarasaba @ElenaStoeva ! I'm mad at myself for not including that to begin with... I actually went back to my prototype to check that 😫 Yes please feel free to add the tooltip! And there is never a situation where an additional aria label wouldn't be welcome. You'd be making our A11y team very happy... and it would make the console a good example of our a11y focus.

Reply:
Added the tooltip and updated the screenshot in the PR!